src/wallet/wallet_accounts.rs

```diff
@@ -405,6 +405,45 @@ mod tests {
         assert_eq!(loaded, private);
     }
 
+    #[test]
+    #[ignore]
```

Comment:
ignored, will not be run, you can use this if you need to generate big profiles.